ddl/reorg.go

```diff
@@ -295,6 +302,29 @@ func (d *ddlCtx) GetTableMaxRowID(startTS uint64, tblInfo *model.TableInfo) (max
 
 var gofailOnceGuard bool
 
+// getPartitionRange gets the start and end handle of a partition (or table).
+func getPartitionRange(d *ddlCtx, tblInfo *model.TableInfo, partitionID int64, snapshotVer uint64) (StartHandle, EndHandle int64, err error) {
```

Comment:
Please make the Table/Partition usage consistent. For other functions, we always use `Table`. For example:
```go
GetTableMaxRowID
buildDescTableScan
constructDescTableScanPB
```